packages/modules/pv/ethmpm3pmaevu.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
```

Comment:
Das müsste doch eigentlich das identische Modul sein, wie das pv/openwb.py, bis auf die IP Adresse. Wenn hinter einem EVU Kit, ist das eben die 192.168.178.15 des EVU Kits und nicht die 192.168.178.13 wenn reines PV Kit.

Reply:
"Work in Progress", sag ich mal.

Reply:
@lkuemmel schau dir mal den PR 1578 flexible PV Module an. Die Benennungen der PVkit Modulen war bisher etwas  inkonsistent.